docs/guides/high-availability/introduction.adoc

```diff
@@ -12,6 +12,101 @@ Those setups are intended for a transparent network on a single site.
 The {project_name} high-availability guide goes one step further to describe setups across multiple sites.
 While this setup adds additional complexity, that extra amount of high availability may be needed for some environments.
 
+== When to use a multi-site setup
+
+The multi-site deployment capabilities of {project_name} are targeted at use cases that:
+
+* Are constrained to a single
+<@profile.ifProduct>
+AWS Region.
+</@profile.ifProduct>
+<@profile.ifCommunity>
+AWS Region or an equivalent low-latency setup.
+</@profile.ifCommunity>
+* Permit planned outages for maintenance.
+* Fit within a defined user and request count.
+* Can accept the impact of periodic outages.
+
+<@profile.ifCommunity>
+== Tested Configuration
+
+We regularly test {project_name} with the following configuration:
+</@profile.ifCommunity>
+<@profile.ifProduct>
+== Supported Configuration
+</@profile.ifProduct>
+
+* Two Openshift single-AZ clusters, in the same AWS Region
+** Provisioned with https://www.redhat.com/en/technologies/cloud-computing/openshift/aws[Red Hat OpenShift Service on AWS] (ROSA),
+<@profile.ifProduct>
+either ROSA HCP or ROSA classic.
+</@profile.ifProduct>
+<@profile.ifCommunity>
+using ROSA HCP.
+</@profile.ifCommunity>
+
+** Each Openshift cluster has all its workers in a single Availability Zone.
+** OpenShift version
+<@profile.ifProduct>
+4.16 (or later).
+</@profile.ifProduct>
+<@profile.ifCommunity>
+4.16.
+</@profile.ifCommunity>
+
+* Amazon Aurora PostgreSQL database
+** High availability with a primary DB instance in one Availability Zone, and a synchronously replicated reader in the second Availability Zone
+** Version ${properties["aurora-postgresql.version"]}
+* AWS Global Accelerator, sending traffic to both ROSA clusters
+* AWS Lambda
+<@profile.ifCommunity>
+triggered by ROSA's Prometheus and Alert Manager
+</@profile.ifCommunity>
+to automate failover
+
+<@profile.ifProduct>
+Any deviation from the configuration above is not supported and any issue must be replicated in that environment for support.
+</@profile.ifProduct>
+<@profile.ifCommunity>
+While equivalent setups should work, you will need to verify the performance and failure behavior of your environment.
+We provide functional tests, failure tests and load tests in the https://github.com/keycloak/keycloak-benchmark[Keycloak Benchmark Project].
```

Comment:
Do we need to link to specific test documentation as well?

Reply:
We currently don't provide docs on how to run the functional tests outside of the environment we're currently using, so it would be hard for another person to replicate it. Once we have the docs, we could link to them. So I'd want to keep it as that for now.

Reply:
Maybe we should point people to releases page in Keycloak benchmark instead of the whole repository and instruct people to find release matching their KC version. Then when doing release in kcb we will document this version work with Keycloak 26.0.x. WDYT?